myapp.rb

```diff
@@ -5,26 +5,49 @@
 require 'sinatra/flash'
 require './models/message'
 require './models/user'
+require './models/follower'
 
 PR_PAGE = 30
 
-DATABASE_PATH = ENV['DATABASE_PATH'] || './db/minitwit.db'
+DATABASE_URL = ENV['DATABASE_URL']
 
 configure :production do
-  set :database, { adapter: 'sqlite3', database: DATABASE_PATH }
+  db = URI.parse(DATABASE_URL)
+  set :database, {
+    adapter: db.scheme,
+    host: db.host,
+    port: db.port,
+    database: db.path[1..],
+    user: db.user,
+    password: db.password,
+    encoding: 'utf8'
+  }
   set :public_folder, "#{__dir__}/static"
   enable :sessions
 end
 
 configure :development do
-  set :database, { adapter: 'sqlite3', database: './db/minitwit_dev.db' }
+  set :database, { adapter: 'postgresql', database: 'minitwit_development' }
```

Comment:
Where is the name `minitwit_development` generated?

Reply:
This is generated by running `bundle exec rake db:create`
It creates the db in accordance with the environment set in APP_ENV